public/javascript/main.js

```diff
@@ -376,5 +376,21 @@ $(function(){
 
           $(this).data('chart', chart);
       }
-  })
+  });
+
+  $("a.download-with-term").on("click", function(e){
```

Comment:
Also, if this a class that’s *only* used to hook JS behaviour onto, it’s good practice to prepend it with `js-`, so that it’s more obvious what the class is for, when people are refactoring code. So that’d make the selector look like: `$('.js-download-with-term')`.